adapters/cisco_ios_xr/cisco_ios_xr_connect.php

```diff
@@ -54,6 +54,61 @@ function cisco_ios_xr_disconnect()
 
 class CiscoIsrsshConnection extends SshConnection
```

Comment:
a detail but the ISR in the class name doesn’t make sense

Reply:
Ok, this was the case in initial delivery. I'll fix it.